src/components/KeypressListener/tests/KeypressListener.test.tsx

```diff
@@ -14,11 +14,11 @@ const listenerMap: HandlerMap = {};
 
 describe('<KeypressListener />', () => {
   beforeEach(() => {
-    document.addEventListener = jest.fn((event, cb) => {
+    jest.spyOn(document, 'addEventListener').mockImplementation((event, cb) => {
```

Comment:
We'll want to change the afterEach to a clear rather than reassign the original value